lib/fathead/sass/parse.py

```diff
@@ -0,0 +1,221 @@
+        return ''
+
+    def create_url(self, anchor):
+        """
+        Helper method to create URL back to document
+        Args:
+            anchor: #anchor
+
+        Returns:
+            Full URL to function on the python doc
+
+        """
+        return PYTHON_DOC_BASE_URL + anchor
+
+    def parse_for_data(self):
+        """
+        Main gateway into parsing the data. Will retrieve all necessary data elements.
+        """
+        data = []
+        
+        for function_section in self.function_sections:
+            function = self.parse_for_function_name(function_section[0])
+            if function:
+                method_signature = self.parse_for_method_signature(function_section[0])
+                description = self.parse_for_description(function_section[1])
+                anchor = self.parse_for_anchor(function_section[0])
+
+                url = self.create_url(anchor)
+
+                data_elements = {
+                    'function': function,
+                    'method_signature': method_signature,
+                    'description': description,
+                    'url': url
+                }
+
+                data.append(data_elements)
+
+        self.parsed_data = data
+
+    def get_data(self):
+        """
+        Get the parsed data.
+        Returns:
+            self.parsed_data: Dict containing necessary data elements
+        """
+        return self.parsed_data
+
+
+class PythonDataOutput(object):
+    """
+    Object responsible for outputting data into the output.txt file
+    """
+    def __init__(self, data):
+        self.data = data
+
+    def create_file(self):
+        """
+        Iterate through the data and create the needed output.txt file, appending to file as necessary.
+
+        """
+        with open('output.txt', 'w') as output_file:
+            for data_element in self.data:
+                if data_element.get('function'):
+                    method_signature = data_element.get('method_signature').encode('utf-8')
+                    description = data_element.get('description').encode('utf-8')
+                    name = data_element.get('function').encode('utf-8')
+
+                    abstract = method_signature + '<br>' +description
+                    url = data_element.get('url').encode('utf-8')
+                    list_of_data = [
+                        name,                       # unique name
+                        'A',                        # type is article
+                        '',                         # no redirect data
+                        '',                         # ignore
+                        '',                         # no categories
+                        '',                         # ignore
+                        '',                         # no related topics
+                        '',                         # ignore
+                        'http://sass-lang.com/',    # add an external link back to SASS home
```

Comment:
this is handled by the IA Page. The external link isn't necessary because it's the same as our Source URL. The live IA will automatically create a "More at" link that points to sass-lang.com 👍